### DISCUSSION TITLE
Official ITF CHAT

### DISCUSSION THREAD

**Discussion by Pog3k** [eclipse/kiso-testing#237] [I would like to discuss something!]
Hey guys, I want to try if we could use this as an official ITF CHAT

**Comment by cody82** [comment]
Hi! 👯

**Comment by sebastianpfischer** [comment]
Since the addition of the `tox`, the test-report are not generated correcty: https://ci.eclipse.org/kiso-testing/job/kiso-testing/job/master/153/testReport/.
@Pog3k, can you please take a look?

**Comment by sebastianpfischer** [comment]
@sebclrsn, can you take a look again at option 2.1 from the decision? Is it complete enough? I would like to finalise and close the decision. Thanks!
